v2/internal/testcommon/samples_tester.go

```diff
@@ -35,6 +35,9 @@ const (
 // Regex to match '/00000000-0000-0000-0000-000000000000/' strings, to replace with the subscriptionID
 var subRegex = regexp.MustCompile("\\/([0]+-?)+\\/")
 
+// An empty GUID, used to replace the subscriptionID and tenantID in the sample files
+var emptyGuid = "00000000-0000-0000-0000-000000000000"
```

Comment:
There might be a GUID/UUID const for this you could use instead?

Reply:
Yup. `uuid.Nil.String()`.